demos/google_api_solution.py

```
@@ -0,0 +1,217 @@
+            
+            if result.generated_ideas:
+                print(f"✅ Completed in {phase_time:.2f}s")
+                print(f"💡 Generated {len(result.generated_ideas)} ideas:")
+                
+                for idea in result.generated_ideas:
+                    print(f"   • {idea.content[:100]}...")
+                    all_ideas.append(idea)
+                
+                # Track cost if available
+                if hasattr(result, 'generation_metadata') and result.generation_metadata:
+                    if 'llm_cost' in result.generation_metadata:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)